wicker/core/column_files.py

```diff
@@ -318,3 +318,6 @@ def process_object_field(self, field: schema.ObjectField) -> Any:
             return data
         cbf_info = ColumnBytesFileLocationV1.from_bytes(data)
         return self.cbf_cache.read(cbf_info)
+
+    def process_sf_variant_field(self, field: schema.VariantField) -> Any:
```

Comment:
do you want to store the reference in the s3 backed column files? I don't think we should store numpy  in the column files, that would generated tons of small files that both kill the data governance and loading performance?